DataScience/Analytics/linking/addressLinking.py

```diff
@@ -698,6 +734,11 @@ def _find_likeliest_address(self, addresses_to_be_linked, blocking=1):
         compare.string('LOCALITY', 'Locality', method='jarowinkler', name='locality_dl',
                        missing_value=0.5)
 
+        # add a comparison of the incode - this helps with e.g. life events addresses
+        if self.settings['expandPostcode']:
+            compare.string('postcode_in', 'postcode_in', method='jarowinkler', name='incode_dl',
```

Comment:
The first two positions define which dataframe columns hold the information being compared. In this case the column names are the same, but one refers to the input dataframe while the other to the AddressBase dataframe.